be/src/storage/tablet_manager.h

```diff
@@ -62,7 +62,7 @@ class RowsetsAcqRel {
 
 private:
     DISALLOW_COPY_AND_MOVE(RowsetsAcqRel);
-    std::vector<RowsetSharedPtr> _rowsets;
+    const std::vector<RowsetSharedPtr>& _rowsets;
```

Comment:
it is very dangerous,  if the reference escape from the scope of object owner.  copy here is not performance-critical